### ISSUE TITLE
[Feature Request] - Improve community contributions

### ISSUE THREAD

**Issue by onmax** [nimiq/crypto-map-old#11]
### Community interaction and feedback

1. **Quick Feedback System:** It's become clear that many of the locations listed on the map are either out of date or incorrect. To address this, it would be beneficial to implement a quick feedback system that allows users to easily report inaccuracies. This would help keep the map accurate and up to date.

2. **Like/Dislike and thermometer feature:** The reintroduction of a like/dislike feature, similar to the old YouTube model, could provide immediate feedback on the reliability of listed locations. In addition, adding a "thermometer" feature could indicate which businesses have recently accepted crypto, giving users a quick way to identify trustworthy places.

3. **Leaderboards and rewards:** Introducing leaderboards to showcase top contributors could motivate more users to participate. A "zap" feature could also be added to allow users to recognise and reward the efforts of trusted editors, fostering a sense of community appreciation.

4. **Streamlined Edit Approvals:** Implement a system where edits from high quality editors are fast-tracked for approval. This would ensure that reliable updates are reflected on the map more quickly, improving the overall user experience.

### Trust, verification and collaboration

1. **Editor verification:** Establish a system that verifies edits based on the editor's history. This would help users trust the information provided and maintain the credibility of the map.

2. **Business verification:** A major problem reported by users is that some businesses listed as accepting crypto don't actually do so. Implementing a verification system where users can confirm that businesses accept crypto would greatly reduce this problem. In addition, allowing businesses to register themselves as crypto-friendly could improve accuracy.

3. **Local Editing Teams:** Forming local editing teams with tools such as dedicated Telegram groups could facilitate real-time discussion and better coordination of edits in specific areas.

### Technical considerations

1. **Authentication system:** Although efforts have been made to avoid using a backend with authentication, it seems necessary to implement one to improve the reliability of the system. Using the Nimiq wallet for authentication could provide a secure and streamlined solution.

### Improve search algorithm

1. Allow users to use keywords or products that the businness might sell. For example: "Chinese Restaurant"

**Comment by Albermonte** [comment]
From what I've seen on the map, there are a lot of places that are either fake or outdated. So, some kind of fast user feedback is necessary imo.
I like the idea of like/dislike, I would add something like a termometer to know which locations are known to be accepting crypto recently or similar.

Community discussion is maybe overkill since we don't have such a big community adding and reporting locations right now

**Comment by onmax** [comment]
Yes, but I avoid using a backend with auth functionality. I guess we will have to implement it 😓.

Maybe auth system with Nimiq Wallet?
